samples/HostedInAspNet.Client/HostedInAspNet.Client.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <BlazorLinkOnBuild>true</BlazorLinkOnBuild>
```

Comment:
Later on if you don't mind, I'll remove the `<BlazorLinkOnBuild>` override from the samples (so they build quicker in development) and just make the E2E tests pass in an override somehow. Will that be OK?

Reply:
Yes. We can do that by passing /p:BlazorLinkOnBuild=true somewhere in the tests I guess. For MonoSanity we need to use TreatPropertyAsLocal to prevent /p:BlazorLinkOnBuild=true from ever overriding